sync/sync_test.go

```diff
@@ -197,102 +231,27 @@ func TestSubscribeNewHeads(t *testing.T) {
 	sub.Unsubscribe()
 }
 
-func TestPendingSync(t *testing.T) {
+func TestSubscribePendingTxs(t *testing.T) {
 	t.Parallel()
 
 	client := feeder.NewTestClient(t, &utils.Mainnet)
 	gw := adaptfeeder.New(client)
 
-	var synchronizer *sync.Synchronizer
 	testDB := pebble.NewMemTest(t)
 	log := utils.NewNopZapLogger()
-	bc := blockchain.New(testDB, &utils.Mainnet, synchronizer.PendingBlock)
-	synchronizer = sync.New(bc, gw, log, time.Millisecond*100, false, testDB)
+	bc := blockchain.New(testDB, &utils.Mainnet, nil)
+	synchronizer := sync.New(bc, gw, log, time.Millisecond*100, false, testDB)
 	ctx, cancel := context.WithTimeout(context.Background(), 3*time.Second)
 
+	sub := synchronizer.SubscribePendingTxs()
+
 	require.NoError(t, synchronizer.Run(ctx))
 	cancel()
 
-	head, err := bc.HeadsHeader()
-	require.NoError(t, err)
 	pending, err := synchronizer.Pending()
 	require.NoError(t, err)
-	assert.Equal(t, head.Hash, pending.Block.ParentHash)
-}
-
-func TestPending(t *testing.T) {
```

Comment:
Why remove the subtest?

Reply:
Do you mean `TestPending`? It wasn't removed, it was moved to a [separate location](https://github.com/NethermindEth/juno/blob/9104a8cc3d06d6a3196796905143763bd1a6d31a/sync/sync_test.go#L188)